### DISCUSSION TITLE
为啥又放开讨论了？

### DISCUSSION THREAD

**Discussion by zhaomingtao-fighting** [WeNeedHome/SummaryOfLoanSuspension#838] [General]
现在不怕了吗？

**Comment by chimission** [comment]
我觉得不如直接关闭讨论，在很多复杂的公共事件讨论上，评论最终会变为只为了怄气的口水仗

**Reply by chosai** [reply]
紫蜡烛能不能老老实实给烂尾楼交贷款？一边停贷一边爱国？又当又立？

**Reply by Sibitmoon** [reply]
感恩🥰🥰🥰

**Reply by chimission** [reply]
> 紫蜡烛能不能老老实实给烂尾楼交贷款？一边停贷一边爱国？又当又立？  

speak of the devil, here he is

**Reply by chosai** [reply]
> > 紫蜡烛能不能老老实实给烂尾楼交贷款？一边停贷一边爱国？又当又立？
> 
> speak of the devil, here he is

She.

**Comment by oceanzhang860411** [comment]
開不開討論區，根本沒有意義，
第一，河南儲戶的事珠玉在前，目前執政者的操作不是解決問題，是分化群體事件的各方，
第二，在整個地產鏈條中，賺的最多的是地方政府，監督的也是地方政府，又是運動員又是裁判員，
第三，目前這個量級的停貸，銀行根本不怕你，在貸款總量裏占比例太低了
都沒有抓住主要問題，討論個J 8

**Reply by ztex** [reply]
是的  毕竟没有统战价值

**Reply by oceanzhang860411** [reply]
有沒有統戰價值其實也是受害者自己決定的，你只是溫和的發發聲明，鐵拳是不會在意的。

**Reply by k0spider** [reply]
> 有沒有統戰價值其實也是受害者自己決定的，你只是溫和的發發聲明，鐵拳是不會在意的。

很有道理。这300+，和恒大对比一下，连零头都达不到。恒大都不会救，更何况这点数量级，上面都睁一只眼闭一只眼，随便回应一下，意思一下有过关注，过去了也就过去了。最终还是会冷处理……

**Reply by oceanzhang860411** [reply]
還有一種惡心的玩法，就是允許貸款人不還款，但是也不退款，房子也跟你沒關係。然後地方政府聯合大國企或者城投公司，繼續發債，把爛尾項目吃下來，將來再包裝出售或者出租，反正拖唄...只要現在不爆，就可以升官

**Reply by ztex** [reply]
和996那个事情是一样的 现在就可能差的是一个催化剂吧 或者点火的吧

**Reply by Accelerator6666** [reply]
> 還有一種惡心的玩法，就是允許貸款人不還款，但是也不退款，房子也跟你沒關係。然後地方政府聯合大國企或者城投公司，繼續發債，把爛尾項目吃下來，將來再包裝出售或者出租，反正拖唄...只要現在不爆，就可以升官

你说的完全有可能变成事实

**Reply by zhw173** [reply]
银行现在不怕不代表以后不怕

**Comment by John-zhan** [comment]
可能吸引不了官老爷关注，需要一波批评指正，来吸引官老爷

**Comment by radiateboy** [comment]
只要媒体不报道就没了热度。

**Comment by volans19** [comment]
因为不讨论更没人关注了，实际上已经没几个人关注了

怕不怕的问题，你要搞清情况
无论这里讨论的多热火朝天，党屁毛都不在乎，他只在乎有没有钱继续奴役老百姓
党的工作重点是挨家挨户让他们交贷款，重要程度远大于这个项目，而这个项目的存在本身的威胁又远远大于讨论区是否开放
开不开讨论真的连党的一根毛都伤不到，而这个项目的统计数字还稍微有点让人知道真相的作用
所以关不关闭只在于有没有自己把自己吓趴下，想不想自我阉割

**Comment by frank822** [comment]
真的不如关闭评论，认认真真的把这个repo做好。看了很多评论，感觉意义并不大，所以，为了避免麻烦，也为了repo能够长久，还不如关闭掉。

**Reply by chimission** [reply]
有很多的评论开始都是很好的切入点可以讨论， 但是随着回复的增加全部都变成了各种人身攻击，看的心累

**Reply by mand-byte** [reply]
看的心累就不看啊，为什么不喜欢的要别人也不喜欢？这么喜欢统一别人的思想吗？

**Reply by oceanzhang860411** [reply]
不能聼不同聲音的蛆，建議還是墻裏呆著，你們的主人也沒允許你們翻出來
